### PR TITLE
feat: Add App Attestation Service

### PR DESCRIPTION
**Pull request by grausof**
Adds the App Attestation Service entity which will be used to obtain the nonce from the Wallet Instance which will be used by the Wallet Provider for the integrity check

## Review

- [X] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [X] English version
- [X] Example files 
- [X] Ask for review

### CODE REVIEW

File: docs/en/wallet-instance-attestation.rst [hunk 1]
```diff
@@ -118,7 +119,7 @@ Payload
 ||       ||                                                            |
 +--------+-------------------------------------------------------------+
 || nonce || The nonce obtained from the                                |
-||       || Wallet Provider.                                           |
+||       || App Attestation Service.                                   |
```
Comment (peppelinux) [inline docs/en/wallet-instance-attestation.rst:122]: Ok, can I ask you to add the App Attestation Service here
https://github.com/italia/eudi-wallet-it-docs/blob/versione-corrente/docs/en/defined-terms.rst#defined-terms

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: docs/en/wallet-instance-attestation.rst (hunk 1 above)
Comment (grausof) [reply docs/en/wallet-instance-attestation.rst:122]: Already added https://github.com/italia/eudi-wallet-it-docs/pull/121/files#diff-0b13b354d3cc69c57166e38bca0c889a5e024419acaed05995f6c1ffe5d3b6e4R40

**Comment by balanza** [comment]
For easy comparison. 

**Before**
image


**After**
image

**Review by balanza — APPROVED** [review]

File: docs/en/defined-terms.rst [hunk 2]
```diff
@@ -37,6 +37,8 @@ Below are the description of acronyms and definitions which are useful for furth
      - All public and/or private entities, conforming to a technical profile and accredited by the Federation Authority, that provide citizens with an IT Wallet Instance.
    * - Wallet Instance Attestation
      - Verifiable Attestation, issued by the Wallet Provider, that proves the security compliace of the Wallet Instance.
+   * - App Attestation Service
+     - External service from the device manufacturer that allows you to certify the authenticity of the mobile app (Wallet Instance).
```
Comment (peppelinux) [inline docs/en/defined-terms.rst]:
```suggestion
     - Device manufacturer service that allows you to certify the authenticity of the mobile app (Wallet Instance).
```

File: docs/en/wallet-instance-attestation.rst [hunk 3]
```diff
@@ -62,12 +62,13 @@ This section describes the Wallet Instance Attestation format and how the Wallet
...
 
 - **Message 1**: The User starts the Wallet Instance mobile app, a new Wallet Instance Attestation is automatically obtained if the previous one results expired.
 - **Message 2-3**: The Wallet Instance retrieves metadata about its Wallet Provider, including the list of supported algorithms, public keys, and endpoints.
 - **Message 4**: The Wallet Instance verifies the Wallet Provider's trustworthiness by resolving the provider's trust chain to the Trust Anchor.
-- **Message 5-7**: The Wallet Instance generates a new key pair and requests a ``nonce`` from the Wallet Provider to guard against replay attacks.
+- **Message 5**: The Wallet Instance generates a new key pair.
+- **Message 6-7**: The wallet instance requests a ``nonce`` from the App Attestation Service.
```
Comment (peppelinux) [inline docs/en/wallet-instance-attestation.rst]:
```suggestion
- **Message 6-7**: The Wallet Instance requests a ``nonce`` from the App Attestation Service.
```

File: docs/en/wallet-instance-attestation.rst [hunk 4]
```diff
@@ -83,7 +84,7 @@ Format of the Wallet Instance Attestation Request
 To obtain a Wallet Instance Attestation from the Wallet
 Provider it is necessary to send a Wallet Instance Attestation
 Request from the Wallet Instance containing the associated public key
-and a ``nonce`` previously requested to avoid replay attacks.
+and a ``nonce`` previously requested. A ``jwti`` is used to avoid replay attacks.
```
Comment (peppelinux) [inline docs/en/wallet-instance-attestation.rst]:
```suggestion
, the ``nonce`` value previously requested and a ``jti`` value.
```

Comment (peppelinux) [reply docs/en/wallet-instance-attestation.rst]: @grausof considering we already have the nonce, is the jti required?
I mean that by just using the nonce the replay attacks can be avoided

**Review by peppelinux — APPROVED** [review]

**Pull request merged**